src/core/Akka/IO/Tcp.cs

```diff
@@ -1011,33 +807,22 @@ private PeerClosed()
         /// </summary>
         public sealed class ErrorClosed : ConnectionClosed
         {
-            /// <summary>
-            /// TBD
-            /// </summary>
-            /// <param name="cause">TBD</param>
             public ErrorClosed(string cause)
             {
                 Cause = cause;
             }
 
-            /// <summary>
-            /// TBD
-            /// </summary>
             public override bool IsErrorClosed => true;
-
-            /// <summary>
-            /// TBD
-            /// </summary>
-            /// <returns>TBD</returns>
-            public override string Cause { get; }
+            
+            public override string? Cause { get; }
```

Comment:
The cause can be null?

Reply:
Not sure about that actually - let me check